src/main/kotlin/com/cout970/modeler/gui/react/leguicomp/Panel.kt

```diff
@@ -33,10 +36,6 @@ open class Panel : LeguiPanel<LeguiComponent>() {
             position.y = y
         }
 
-    operator fun LeguiComponent.unaryPlus() {
```

Comment:
Why do you remove this?
I like to put `+Button(...)` instead of `add(Button(...))`